apps/rehabstod/src/components/Table/tableBody/TableRow.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactNode } from 'react'
+
+export function TableRow({
+  italic,
+  onClick,
+  id,
+  children,
+}: {
+  italic: boolean
+  onClick: (key: string) => void
```

Comment:
onClick sounds like we would be expecting a mouse event. Perhaps rename to onSelect, onNavigate or something 🤷‍♂️